requirements-dev.txt

```
@@ -1,13 +1,11 @@
 ophyd
-
-# These are required for developing the package (running the tests, building
-# the documentation) but not necessarily required for _using_ it.
+msgpack>=1.0.0
```

```suggestion
msgpack >=1.0.0
```
Conda requires a space here. Pip doesn't care, but if we use a space it's easier to sync between this and the conda recipe.

---

done